homeassistant/components/private_ble_device/coordinator.py

```diff
@@ -102,6 +102,18 @@ def _async_track_unavailable(
 
     def _async_irk_resolved_to_mac(self, irk: bytes, mac: str) -> None:
         if previous_mac := self._irk_to_mac.get(irk):
+            previous_interval = bluetooth.async_get_learned_advertising_interval(
+                self.hass, previous_mac
+            )
+            if not previous_interval:
+                previous_interval = bluetooth.async_get_fallback_availability_interval(
+                    self.hass, previous_mac
+                )
```

Comment:
```suggestion
            previous_interval = bluetooth.async_get_learned_advertising_interval(
                self.hass, previous_mac
            ) or bluetooth.async_get_fallback_availability_interval(
                self.hass, previous_mac
            )
```

I guess we don't care which value it is since we are going to set it for the fallback?

Reply:
Yep I think that makes sense.